metadata.rb

```diff
@@ -1,19 +1,11 @@
 name              'mongodb'
-maintainer        'edelight GmbH'
-maintainer_email  'markus.korn@edelight.de'
+maintainer        'Chef Brigade '
+maintainer_email  'help@chefbrigade.io'
 license           'Apache 2.0'
 description       'Installs and configures mongodb'
+long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version           '0.16.3'
 
-recipe 'mongodb', 'Installs and configures a single node mongodb instance'
-recipe 'mongodb::10gen_repo', 'Adds the 10gen repo to get the latest packages'
-recipe 'mongodb::mongos', 'Installs and configures a mongos which can be used in a sharded setup'
-recipe 'mongodb::configserver', 'Installs and configures a configserver for mongodb sharding'
-recipe 'mongodb::shard', 'Installs and configures a single shard'
-recipe 'mongodb::replicaset', 'Installs and configures a mongodb replicaset'
-recipe 'mongodb::mms_monitoring_agent', 'Installs and configures a MongoDB MMS Monitoring Agent'
-recipe 'mongodb::mms_backup_agent', 'Installs and configures a MongoDB MMS Backup Agent'
```

Comment:
I don't know if it is on purpose, but you have removed all the `recipe` and `attribute` documentation settings from the metadata.